datalad/interface/tests/test_run.py

```diff
@@ -14,54 +14,72 @@
 
 import logging
 
-
+import os
 import os.path as op
-from os.path import join as opj
-from os.path import relpath
-from os import mkdir, remove
+from os import (
+    mkdir,
+    remove,
+)
 import sys
 
 from six import PY2
 from six.moves import StringIO
 from mock import patch
 
-from datalad.utils import assure_unicode
-from datalad.utils import chpwd
+from datalad.utils import (
+    assure_unicode,
+    chpwd,
+    on_windows,
+)
 
 from datalad.cmdline.main import main
 from datalad.distribution.dataset import Dataset
-from datalad.support.exceptions import NoDatasetArgumentFound
-from datalad.support.exceptions import CommandError
-from datalad.support.exceptions import IncompleteResultsError
 from datalad.support.gitrepo import GitRepo
-from datalad.tests.utils import ok_, assert_false, neq_
-from datalad.api import install
-from datalad.api import run
-from datalad.interface.run import format_command
-from datalad.interface.run import run_command
-from datalad.interface.rerun import get_run_info
-from datalad.interface.rerun import diff_revision, new_or_modified
-from datalad.tests.utils import assert_raises
-from datalad.tests.utils import assert_dict_equal
-from datalad.tests.utils import with_tempfile
-from datalad.tests.utils import with_tree
-from datalad.tests.utils import ok_clean_git
-from datalad.tests.utils import ok_exists
-from datalad.tests.utils import ok_file_under_git
-from datalad.tests.utils import ok_file_has_content
-from datalad.tests.utils import create_tree
-from datalad.tests.utils import eq_
-from datalad.tests.utils import assert_status
-from datalad.tests.utils import assert_result_count
-from datalad.tests.utils import assert_in
-from datalad.tests.utils import assert_not_in
-from datalad.tests.utils import swallow_logs
-from datalad.tests.utils import swallow_outputs
-from datalad.tests.utils import assert_in_results
-from datalad.tests.utils import known_failure_windows
-from datalad.tests.utils import slow
-from datalad.tests.utils import with_testrepos
-from datalad.tests.utils import OBSCURE_FILENAME
+from datalad.support.exceptions import (
+    NoDatasetArgumentFound,
+    CommandError,
+    IncompleteResultsError,
+)
+from datalad.api import (
+    install,
+    rev_run as run,
+)
+from datalad.interface.run import (
+    format_command,
+    run_command,
+)
+from datalad.interface.rerun import (
+    get_run_info,
+    diff_revision,
+    new_or_modified,
+)
+from datalad.tests.utils import (
+    assert_raises,
+    assert_false,
+    assert_dict_equal,
+    assert_in_results,
+    assert_repo_status,
+    with_tempfile,
+    with_tree,
+    ok_,
+    ok_exists,
+    ok_file_has_content,
+    ok_file_under_git,
+    create_tree,
+    eq_,
+    neq_,
+    assert_status,
+    assert_result_count,
+    assert_in,
+    assert_not_in,
+    swallow_logs,
+    swallow_outputs,
+    known_failure_windows,
+    slow,
+    with_testrepos,
+    OBSCURE_FILENAME,
+    SkipTest,
+)
```

Comment:
May be sort them while at it to minimize future conflicts someone decided to do so?

It these all changes come from revolution as is/with minimal changes and thus would be more preferable for future changes ingestion (but should there be more for the run)?

Reply:
These changes are coming from -revolution's side and just retain the order that was used there.  The merge commit is here for you to inspect: 3b3818c 

This is a one-time merge.